stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -30,7 +30,7 @@ open class StripeEditText @JvmOverloads constructor(
 
     private var afterTextChangedListener: AfterTextChangedListener? = null
     private var deleteEmptyListener: DeleteEmptyListener? = null
-    var cachedColorStateList: ColorStateList? = null
+    var cachedColorStateList: ColorStateList
```

Comment:
I think we should call this `defaultColor`, to make default and error complement each other.